ansible/update_reboot.yml

```diff
@@ -8,7 +8,6 @@
   become: true
 
 - name: reboot vm_host
-  reboot: reboot_timeout=600
+  reboot: reboot_timeout=900
   when: '"0 upgraded" not in apt_update_res.stdout'
   become: true
-
```

Comment:
Maybe need a newline?

Reply:
Originally the file has two new lines. I removed the extra one. Now there is indeed a single new line by the end of file.